core/src/main/java/edu/wpi/grip/core/settings/CodeGenerationSettings.java

```diff
@@ -0,0 +1,160 @@
+package edu.wpi.grip.core.settings;
```

Comment:
Not your problem but I'm going to complain anyways:
This is so much boilerplate. Would be much shorter in kotlin.

You don't need to do anything to change it though.